Test/DoubleProtease.tsv

```diff
@@ -1,4 +1,4 @@
 Name	Sequences Inducing Cleavage	Sequences Preventing Cleavage	Cleavage Terminus	Cleavage Specificity	PSI-MS Accession Number	PSI-MS Name	Site Regular Expression	Notes
-Test1	K|R|D	"P,D|P|"	C|N|N	full				
-Test2	K|D		C|C	full				
-Test3	K|		C|N	full				
+Test1	"K|,|R,|D[D]"			full				
```

Comment:
is this the same protease as before? shouldn't it be
`K|[P],K|[D],|R[P],|D` ?

Reply:
so i took a look at the test and the protein OKAREDY would be digested into: OK, A, REDY under Test 1.
so maybe the protease would be K|[P],K|[D],|R[P],|D**[D]** ?